appointment_report.Rmd

```diff
@@ -0,0 +1,83 @@
+---
+title: "First Responder Covid-19 Report"
+date: "`r Sys.Date()`"
+output: pdf_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(
+	echo = FALSE,
+	message = FALSE,
+	warning = FALSE
+)
+
+library(tidyverse)
+library(dotenv)
+library(REDCapR)
+library(lubridate)
+library(kableExtra)
+
+# set the timezone
+Sys.setenv(TZ = Sys.getenv("TIME_ZONE"))
+
+project_start_date <- ymd("2020-04-08")
+current_date <- Sys.Date() - 1
+```
+
+Since we do not specifically track ‘No-Show’ appointments for this study we are operating under the assumption that if the appointment occurred and 48 hours elapsed without a test result being entered then the subject did not show up for their appointment and thus had no sample collected.
+
+
+```{r}
+# read appointment data betewwen 2020-04-08 and current_date 
+records <- redcap_read_oneshot(redcap_uri = 'https://redcap.ctsi.ufl.edu/redcap/api/',
+                               token = Sys.getenv("FR_API_TOKEN"),
```

Comment:
```suggestion
                               token = Sys.getenv("TOKEN"),
```
As we make more and more of these environment-configured containers, we'd be well-served to standardize on names for env vars.